schedule/yam/test_cases/textmode_installation_minimal_role.yaml

```diff
@@ -6,11 +6,9 @@ description:    >
     2. "Minimal" role is selected;
     3. Boot to command-line mode;
     4. Installation is validated by default set of smoke tests.
-vars:
-  YUI_REST_API: 1
 schedule:
-  system_role:
-    - installation/system_role/select_role_minimal
+  default_systemd_target:
+    - installation/installation_settings/validate_default_target
```

Comment:
you cannot remove minimal role:
https://openqa.suse.de/tests/13084484#step/select_role_minimal/1
(there is a visual bug in the libyui-rest-api that both look selected, but you can see later that is not the case)

Reply:
Thanks for your remind,  Update the file.